### DISCUSSION TITLE
chia help me

### DISCUSSION THREAD

**Discussion by MuchaCaelan284** [Chia-Network/chia-blockchain#6635] [Support]
Why can't I connect to my wallet?
![chia-problem](https://user-images.githubusercontent.com/85616243/121334891-c10f2b80-c922-11eb-989d-a8b3660403cd.png)

**Comment by loppefaaret** [comment]
it can be a number of things, but generally, it is mostly from a corrupted wallet databse, if the client has not been closed down gracefully.
to try and fix this, close down the Chia app, and open up a PowerShell window, and execute this command below:
`mv ~\.chia\mainnet\wallet\db\block* ~\Desktop`
this will move the wallet database from the working folder, to your desktop
now try opening up the Chia app again - and if this is now successfully opening, you had a corrupted wallet database, and can delete the file(s) on your desktop, and sync the wallet from 0

**Reply by MuchaCaelan284** [reply]
Thank you for your reply, but unfortunately the problem persists. :(

**Reply by loppefaaret** [reply]
have you had the app working prior to having this issue ?

**Reply by MuchaCaelan284** [reply]
Yes my chia app is working and farming 2 days ago  I see when my wallet is not synced and I delete chia after reinstall but screen is always loading

**Reply by loppefaaret** [reply]
in that case, i would perhaps try a full reset:
close down Chia app, and open up PowerShell, and execute below commands:
```
mv ~\.chia\mainnet ~\.chia\old
cd ~\AppData\Local\chia-blockchain\app-1.1.7\resources\app.asar.unpacked\daemon\
.\chia.exe init
cd ~\
mv ~\.chia\old\db ~\.chia\mainnet\
```
then try an open up Chia app again - you will have lost all settings, and need to add your plot folders back, and any other settings you need

**Comment by BrandtH22** [comment]
@hajika94 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
